docs/guides/unbound.md

```diff
@@ -52,82 +52,154 @@ The first thing you need to do is to install the recursive DNS resolver:
-
-    # Trust glue only if it is within the servers authority
-    harden-glue: yes
-
-    # Require DNSSEC data for trust-anchored zones, if such data is absent, the zone becomes BOGUS
-    harden-dnssec-stripped: yes
-
-    # Don't use Capitalization randomization as it known to cause DNSSEC issues sometimes
-    # see https://discourse.pi-hole.net/t/unbound-stubby-or-dnscrypt-proxy/9378 for further details
-    use-caps-for-id: no
-
-    # Reduce EDNS reassembly buffer size.
-    # Suggested by the unbound man page to reduce fragmentation reassembly problems
-    edns-buffer-size: 1472
-
-    # Perform prefetching of close to expired message cache entries
-    # This only applies to domains that have been frequently queried
-    prefetch: yes
-
-    # One thread should be sufficient, can be increased on beefy machines. In reality for most users running on small networks or on a single machine it should be unnecessary to seek performance enhancement by increasing num-threads above 1.
-    num-threads: 1
-
-    # Ensure kernel buffer is large enough to not lose messages in traffic spikes
-    so-rcvbuf: 1m
-
-    # Ensure privacy of local IP ranges
+    # Access control to prevent access from non local network devices
+    # Block all
+    access-control: 0.0.0.0/0 refuse
+    access-control: ::0/0 refuse
+    # Allow lo0
+    access-control: 127.0.0.0/8 allow
+    access-control: ::1 allow
+    # Allow lan
+    access-control: 192.168.0.0/16 allow
+    access-control: 172.16.0.0/12 allow
+    access-control: 10.0.0.0/8 allow
+    access-control: fd80:1fe9:fcee::/48 allow
+
+    # Prints one line per query to the log, making the server (significantly) slower
+    #log-queries: no
+
+    # Prevent banner disclosure
+    hide-identity: yes
+    hide-version: yes
+    # Refuse trustanchor.unbound queries
+    hide-trustanchor: yes
+
+    # The maximum dependency depth that Unbound will pursue in answering a query (reduced memory)
+    target-fetch-policy: "2 1 0 0 0 0"
+
+    harden-short-bufsize: yes
+    harden-large-queries: yes
+
+    # Use 0x20-encoded random bits in the query to foil spoof attempts
+    use-caps-for-id: yes
+
+    # DNSSEC bogus answers against DNS Rebinding
+    # RFC1918 private IP address space not allowed to be returned for public internet names
     private-address: 192.168.0.0/16
-    private-address: 169.254.0.0/16
     private-address: 172.16.0.0/12
     private-address: 10.0.0.0/8
+    private-address: 169.254.0.0/16
     private-address: fd00::/8
     private-address: fe80::/10
+    # Turning on 127.0.0.0/8 would hinder many spamblocklists as they use that
+    #private-address: 127.0.0.0/8
+    # Adding ::ffff:0:0/96 stops IPv4-mapped IPv6 addresses from bypassing the filter
+    private-address: ::ffff:0:0/96
+    # add 
+    private-address: 2001:470:b35c::/48
+
+    unwanted-reply-threshold: 10000000
+
+    do-not-query-localhost: no
+
+    # Affects buffer space
+    #prefetch: no
```

Comment:
```diff
-    # Perform prefetching of close to expired message cache entries
-    # This only applies to domains that have been frequently queried
-    prefetch: yes
+    # Affects buffer space
+    #prefetch: no
```
Please add some more explanation for why you would like to see this changed? It can drastically reduce latency of the resolver and comes at fairly low costs.

Reply:
According to the documentation is increases load on the device:

> Turning it on gives about 10 percent more traffic and load on the machine, but popular items do not expire from the cache.

I couldn't find any benchmarks of this with a RPi.